api.js

```diff
@@ -332,6 +337,91 @@
   Api.prototype = {
 
     /**
+     *  HQ Events
+     */
+
+    Events: {
+      NOTIFICATIONS: 'notifications',
+      CUSTOMER     : 'user',
+      MESSAGES     : 'message'
+    },
+
+    on: function (event, func) {
+      if (!this.eventHandlers) {
+        // events not initialized, just return
+        return;
+      }
+
+      if (typeof func !== 'function') {
+        throw 'Not a valid function';
+      }
+
+      // If the event is not yet subscribed to, add it, and listen for it
+      if (!this.eventHandlers[event]) {
```

Comment:
event is a reserved keyword, this can break in some browsers or in the future. Please use something like eventName.